adapters/coap-vertx/src/main/java/org/eclipse/hono/adapter/coap/vertx/Config.java

```diff
@@ -87,6 +93,20 @@ public CoapAdapterProperties adapterProperties() {
         return new CoapAdapterProperties();
     }
 
+    /**
+     * Customizer for meter registry.
+     * 
+     * @return The new meter registry customizer.
+     */
+    @Bean
+    public MeterRegistryCustomizer<MeterRegistry> commonTags() {
+        return r -> r.config().commonTags(
+                Tags.of(
+                        Tag.of("host", Hostnames.getHostname()),
+                        Tag.of("component", "adapter"),
+                        Tag.of("protocol", "coap")));
```

Comment:
IMHO these tag names should be defined as constants (e.g. in MicrometerBasedMetrics ?) so that we can be sure to always use the same values, don't you think?

Reply:
Absolutely. I put it alongside `MicrometerBasedMetrics` though, to make it less coupled and more visible.